packages/contracts/tests/js/ConnectContractsTest.js

```diff
@@ -36,11 +37,12 @@ contract('Deployment script - Sets correct contract addresses dependencies after
 
     zeroStaking = ZEROContracts.zeroStaking;
     zeroToken = ZEROContracts.zeroToken;
+    sovToken = ZEROContracts.zeroToken;
```

Comment:
We should be able to rename the zeroToken, but it will lead us a huge changes for almost all of the unit test since this is the base helper function for the test initialization of deployment.
So i'd like to do it in separate PR, easier for reviewing it as well